### PR TITLE
added a monkey animation

### PR DESCRIPTION
**Pull request by daniel1david**
Hi, please add my contribution. 
Thank you!

### CODE REVIEW

File: Art/daniel1david/art_style.css [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+.box {
+	position: relative;
+	margin: auto;
+	display: block;
+	margin-top: 8%;
+	width: 600px;
+	height: 420px;
+/*	background: white;*/
```
Comment (LaurelineP) [inline Art/daniel1david/art_style.css:14]: # Context
```js
/*	background: white;*/
```

# Feedback - unnecessary comment
Please remove the un used comment

File: Art/daniel1david/art_style.css [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  top: -3%;
+  left: -5%;
+  height: 0;
+  width: 110%;
+  border-radius: 50%;
+  background: saddlebrown;
+  -webkit-animation: blink 3s infinite ease-in;
+  animation: blink 3s infinite ease-in;
+   z-index: 4;
+}
```
Comment (LaurelineP) [inline Art/daniel1david/art_style.css:95]: # Context
```js
.left-eye::after {
  content: "";
  position: absolute;
  top: -3%;
  left: -5%;
  height: 0;
  width: 110%;
  border-radius: 50%;
  background: saddlebrown;
  -webkit-animation: blink 3s infinite ease-in;
  animation: blink 3s infinite ease-in;
   z-index: 4;
}
```

# Feedback - stay consistent
Please try to remain consistent across your own code ( previous block of code has a different indentation ) - 
this should be indented in the same way

File: Art/daniel1david/art_style.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+  top: -3%;
+  left: -5%;
+  height: 0;
+  width: 110%;
+  border-radius: 50%;
+  background: saddlebrown;
+  -webkit-animation: blink 3s infinite ease-in;
+  animation: blink 3s infinite ease-in;
+  z-index: 4;
+}
```
Comment (LaurelineP) [inline Art/daniel1david/art_style.css:120]: # Context
```js
.right-eye::after {
  content: "";
  position: absolute;
  top: -3%;
  left: -5%;
  height: 0;
  width: 110%;
  border-radius: 50%;
  background: saddlebrown;
  -webkit-animation: blink 3s infinite ease-in;
  animation: blink 3s infinite ease-in;
  z-index: 4;
}
```

# Feedback - stay consistent ( indentation ) 
Please try to remain consistent across your own code ( previous block of code has a different indentation ) - 
this should be indented in the same way

File: Art/daniel1david/art_style.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+	height: 10%;
+	left: 30%;
+	top: 62.5%;
+	border-bottom: 5px solid brown;
+	border-radius: 50%;
+	z-index: 4;
+}
+
+/*help from 'https://blog.hubspot.com/website/css-animation-examples#:~:text=To%20make%20a%20CSS%20animation,animation%2C%20like%20speed%20and%20delay.'*/
+
```
Comment (LaurelineP) [inline Art/daniel1david/art_style.css:174]: # Context
```js
/*help from 'https://blog.hubspot.com/website/css-animation-examples#:~:text=To%20make%20a%20CSS%20animation,animation%2C%20like%20speed%20and%20delay.'*/
```

# Feedback - unnecessary comment
Please remove the un used comment

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Thanks for your contribution 🙌
Please address the feedbacks above in order to proceed your PR

**Comment by daniel1david** [comment]
I made the changes to those files. 
Shocked that I didn't have the same styling throughout the whole file; thanks for spotting that!
@LaurelineP

**Pull request merged**